src/skeleton/Skeleton.tsx

```diff
@@ -80,14 +81,14 @@ const Skeleton = (props: SkeletonProps) => {
   const renderRowCol = (_rowCol?: SkeletonRowCol) => {
     const renderedRowCol: SkeletonRowCol = _rowCol || rowCol;
 
-    return renderedRowCol.map((item, index) => (
+    return renderedRowCol.map<React.ReactNode>((item, index) => (
       <div key={index} className={`${name}__row`}>
         {renderCols(item)}
       </div>
     ));
   };
 
-  const [ctrlLoading, setCtrlLoading] = useState(loading);
+  const [ctrlLoading, setCtrlLoading] = useState<boolean>(loading);
```

Comment:
这里的 `loading` 类型应该是能被 `useState` 推断出来的？这里是需要排除掉 `undefined` 的类型吗？
```suggestion
  const [ctrlLoading, setCtrlLoading] = useState(loading);
```

Reply:
对，因为 `TdSkeletonProps['loading']` 是可选的，显式声明一下 boolean 会严谨一点

Reply:
> 对，因为 `TdSkeletonProps['loading']` 是可选的，显式声明一下 boolean 会严谨一点

`skeletonDefaultProps` 存在 loading 的默认值吗，`useDefaultProps` 能不能根据 defaultProps 中的值从而推断出 props 的 `loading` 是必然存在的，如果不可以的话是不是可以加个 TODO 回头可以加个工作继续做一下这块。

Reply:
哦，看了一下 `skeletonDefaultProps` 里面是有 `loading` 默认值的，这样的话我把类型去掉吧

Reply:
update~